### DISCUSSION TITLE
Content type HTML

### DISCUSSION THREAD

**Discussion by dominictobias-bullish** [elysiajs/elysia#534] [Q&A]
I'm quite confused about content types: https://elysiajs.com/life-cycle/parse.html#explicit-content-type

Seems we can only return some, and a glaringly missing one is `text/html`. Can I just set `type` to `"text/html; charset=utf8"`?

I also tried in headers but Typescript doesn't like it:

![Screenshot 2024-03-15 at 11 13 22](https://github.com/elysiajs/elysia/assets/160451465/c2b97b51-efb9-4192-abad-0c5848624b82)

`type` to html doesn't work, the `content-type` becomes `"text/plain;charset=utf-8"`

**Comment by dominictobias-bullish** [comment]
I can get `set` function in the handler and set header with `set.headers['content-type'] = 'text/html; charset=utf8'`

https://elysiajs.com/integrations/cheat-sheet.html#header-and-status-code

**Comment by vitalijalbu** [comment]
Is there a way guys to set JSON header globally??
